src/databricks/labs/ucx/assessment/crawlers.py

```diff
@@ -39,32 +42,56 @@ def _azure_sp_conf_present_check(config: dict) -> bool:
     return False
 
 
-def spark_version_compatibility(spark_version: str | None) -> str:
+@dataclass
+class DBRVersion:
+    major: int
+    minor: int
+
+    def __le__(self, other: DBRVersion):
+        return (self.major, self.minor) <= (other.major, other.minor)
+
+    def __lt__(self, other: DBRVersion):
+        return (self.major, self.minor) < (other.major, other.minor)
+
+
+@dataclass
+class DBRVersionInfo:
+    dbr_version: DBRVersion | None
+    supports_uc: bool
+
+    @property
+    def requires_submit_runs_check(self) -> bool:
```

Comment:
This method is called only once - what is it doing in a module used by many places? This refactoring tightly couples this class with submit runs crawler.

Revert changes to this file. This has to be a private method of submit runs crawler.

Reply:
Removed this property from here, agree it's irrelevant.